elasticsearch/templates/statefulset.yaml

```
@@ -183,7 +187,11 @@ spec:
           {{- end }}
 {{ end }}
       {{- if .Values.extraInitContainers }}
-{{ tpl .Values.extraInitContainers . | indent 6 }}
+      {{- if eq "string" (printf "%T" .Values.extraInitContainers) }}
+{{ tpl .Values.extraInitContainers . | indent 8 }}
+      {{- else }}
+{{ toYaml .Values.extraInitContainers | indent 8 }}
```

```suggestion
{{ tpl .Values.extraInitContainers . | indent 6 }}
      {{- else }}
{{ toYaml .Values.extraInitContainers | indent 6 }}
```

This is what makes https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+template-testing/527/CHART=elasticsearch,label=docker&&virtual/console fail